src/components/data-exploration/heatmap/HeatmapPlot.jsx

```diff
@@ -27,6 +27,8 @@ import HeatmapTracksCellInfo from 'components/data-exploration/heatmap/HeatmapTr
 import getCellClassProperties from 'utils/cellSets/getCellClassProperties';
 import useConditionalEffect from 'utils/customHooks/useConditionalEffect';
 
+import { reversed } from 'utils/arrayUtils';
```

Comment:
I don't see where you use this `reversed`

Reply:
In `populateHeatmapData`, when we call `generateVegaData`